ui/src/components/Tasks/DailyTasks/index.tsx

```diff
@@ -107,23 +153,71 @@ const DailyTaskRow: React.FC<DailyTask & { isFetching: boolean }> = ({ isFetchin
     return day.fromNow();
   };
 
+  const [isSlideBarOpen, setSlideBar] = useState<boolean>(false);
+  const [isDirtyPopupOpen, setIsDirtyPopupOpen] = useState<boolean>(false);
+  const isDirtyRef = useRef<boolean>(false);
+
+  const handleMoreOptions = (item: { id: string; name: string }) => {
+    if (item.id === 'edit') {
+      setSlideBar(true);
+      return;
+    }
+    setModalOpen(true);
+    setModalMessage({
+      subTitle: 'You want to delete this task',
+      title: 'Are you sure ?',
+    });
+    return;
+  };
+
+  const handleCloseSlideBar = () => {
+    if (isDirtyRef.current && !isDirtyPopupOpen) {
+      setIsDirtyPopupOpen(true);
+      return;
+    }
+
+    setSlideBar(false);
+    setIsDirtyPopupOpen(false);
+  };
+  const editValue = {
+    description: p.description || '',
+    estimated_hours: p.estimated_hours || 0,
+    issue_id: p.issue_id,
+    pr_id: p.pr_id,
+    project_id: p.project_id,
+    title: p.title,
+  };
+
   return (
     <>
       <WarningModal
-        onAccept={goToProjectSettings}
+        onAccept={handleOnAccept}
         onCancel={() => setModalOpen(false)}
-        acceptButtonText={'Setting'}
+        acceptButtonText={modalMessage.title === 'warning' ? 'Setting' : 'Ok'}
```

Comment:
1. This check seems brittle. `warning` is quite ambiguous, we shouldn't be making any decisions using that as a basis. 
2. Code above sets the title as `Warning` (capital `W`).  This check will likely never be successful.
3. The accept button text in case of unconfigured pr/issue template should be `Settings` (with an `s).